widgets/memory.go

```diff
@@ -0,0 +1,74 @@
+package widgets
+
+import (
+	"fmt"
+	tui "github.com/gizak/termui/v3"
+	tWidgets "github.com/gizak/termui/v3/widgets"
+	"io/ioutil"
+	"strings"
+)
+
+type MemoryWidget struct {
+	information map[string]int
+	gauge       *tWidgets.Gauge
+}
+
+func NewMemoryWidget() Widget {
+	gauge := tWidgets.NewGauge()
+	gauge.Title = "Memory Usage"
+	gauge.Percent = 0
+	gauge.BarColor = tui.ColorRed
+	gauge.BorderStyle.Fg = tui.ColorWhite
+	gauge.TitleStyle.Fg = tui.ColorCyan
+
+	return &MemoryWidget{
+		information: make(map[string]int),
+		gauge:       gauge,
+	}
+}
+
+func (widget *MemoryWidget) Update() {
+	information := readMemoryInformation()
+	widget.information = information
```

Comment:
값을 바로 대입해도 코드 읽는데는 문제 없을 것 같아요.
```suggestion
	widget.information = readMemoryInformation()
```